blocks_vertical/data.js

```diff
@@ -0,0 +1,167 @@
+'use strict';
+
+goog.provide('Blockly.Blocks.data');
+
+goog.require('Blockly.Blocks');
+
+goog.require('Blockly.Colours');
+
+goog.require('Blockly.constants');
+
+Blockly.Blocks['data_variablemenu'] = {
+  /**
+   * Variable menu.
+   * @this Blockly.Block
+   */
+  init: function() {
+    this.jsonInit(
+      {
+        "message0": "%1",
+        "args0": [
+          {
+            "type": "field_variable",
+            "name": "VARIABLE",
+            "options": [

```

Comment:
I think you can remove this options list, since field_variable populates its own options.